examples/01-filter/image-fft-perlin-noise.py

```diff
@@ -0,0 +1,206 @@
+grid['scalars'] = high_pass['scalars'] + low_pass['scalars']
+
+print(
+    'Low and High Pass identical to the original:', np.allclose(grid['scalars'], sampled['scalars'])
+)
+
+pl = pv.Plotter(shape=(1, 2))
+pl.add_mesh(sampled.warp_by_scalar(), show_scalar_bar=False, lighting=False)
+pl.add_text('Original Dataset')
+pl.subplot(0, 1)
+pl.add_mesh(grid.warp_by_scalar(), show_scalar_bar=False, lighting=False)
+pl.add_text('Sum of the Low and High Passes')
+pl.show()
+
+
+###############################################################################
+# Animate
+# ~~~~~~~
+# Animate the variation of the cutoff frequency.
+
+
+def warp_low_pass_noise(cfreq, scalar_ptp=abs(sampled['scalars']).ptp()):
+    """Process the sampled FFT and warp by scalars."""
+    output = sampled_fft.low_pass(cfreq, cfreq, cfreq).rfft()
+
+    # on the left: raw FFT magnitude
+    output['scalars'] = output.active_scalars.real
+    warped_raw = output.warp_by_scalar()
+
+    # on the right: scale to fixed warped height
+    output_scaled = output.translate((-11, 11, 0), inplace=False)
+    output_scaled['scalars_warp'] = output['scalars'] / abs(output['scalars']).ptp() * scalar_ptp
+    warped_scaled = output_scaled.warp_by_scalar('scalars_warp')
+    warped_scaled.active_scalars_name = 'scalars'
+    # push center back to xy plane due to peaks near 0 frequency
+    warped_scaled.translate((0, 0, -warped_scaled.center[-1]), inplace=True)
+
+    return warped_raw + warped_scaled
+
+
+# Initialize the plotter and plot off-screen to save the animation as a GIF.
+plotter = pv.Plotter(notebook=False, off_screen=True)
+plotter.open_gif("low_pass.gif", fps=8)
+
+# add the initial mesh
+init_mesh = warp_low_pass_noise(1e-2)
+plotter.add_mesh(init_mesh, show_scalar_bar=False, lighting=False, n_colors=128)
+plotter.camera.zoom(1.3)
+
+for freq in np.geomspace(1e-2, 10, 25):
+    plotter.clear()
+    mesh = warp_low_pass_noise(freq)
+    plotter.add_mesh(mesh, show_scalar_bar=False, lighting=False, n_colors=128)
+    plotter.add_text(f"Cutoff Frequency: {freq:.2f}", color="black")
+    plotter.write_frame()
+
+# write the last frame a few times to "pause" the gif
+for _ in range(10):
+    plotter.write_frame()
+
+plotter.close()
+
+
+###############################################################################
+# The left mesh in the above animation warps based on the raw values of the FFT
+# amplitude. This emphasizes how taking into account more and more frequencies
+# as the animation progresses, we recover a gradually larger proportion of the
+# full noise sample. This is why the mesh starts "flat" and grows larger as the
+# frequency cutoff is increased.
+#
+# In contrast, the right mesh is always warped to the same visible height,
+# irrespective of the cutoff frequency. This highlights how the typical
+# wavelength (size of the features) of the Perlin noise decreases as the
+# frequency cutoff is increased, since wavelength and frequency are inversely
+# proportional.
```

Comment:
```suggestion
# In contrast, the right mesh is always warped to the same visible height,
# irrespective of the cutoff frequency. This highlights how the typical
# wavelength (size of the features) of the Perlin noise decreases as the
# frequency cutoff is increased since wavelength and frequency are inversely
# proportional.
```